tests/testthat/_snaps/demo/demo.md

```diff
@@ -0,0 +1,694 @@
+---
+title: "Using fledge"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{demo}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Introduction
+
+This guide will demonstrate how to use {fledge}, using a mock R package as an example.
+We are going to call our package "{tea}".
+We will develop it from scratch and also maintain a changelog as the development progresses.
+Finally, we will demonstrate how this changelog can eventually be converted to release notes when the package is submitted to CRAN.
+
+:::{.alert .alert-info}
+We are typing this demo as an R Markdown vignette therefore we will be using R tools for creating files, editing them, and interacting with git: in real life you can be using e.g. an IDE or the command line. The fledge package won't care!
+:::
+
+
+
+## Set up the development environment
+
+Before we start development for {tea}, we set up the basic development environment required for any typical R package.
+
+### Create a package
+
+We will start by creating a new package.
+For this demo, the package is created in a temporary directory.
+A real project will live somewhere in your home directory.
+
+The `usethis::create_package()` function sets up a package project ready for development.
+
+
+```r
+pkg <- usethis::create_package("tea")
+```
+
+
+
+In an interactive RStudio session, a new window opens and you would work there.
+Users of other environments would change the working directory manually.
+For this demo, we manually set the active project.
+
+
+
+
+```r
+usethis::proj_set()
+## [32mv[39m Setting active project to [34m'${TEMP}/fledge/tea'[39m
+```
+
+The infrastructure files and directories that comprise a minimal R package are created:
+
+
+```r
+fs::dir_tree()
+## [01;34m.[0m
```

Comment:
Should we get rid of the ANSI codes here?

Reply:
yeah I'm on it, good point. :sweat_smile: